src/device-uptime/requirements.txt

```diff
@@ -4,3 +4,5 @@ pymongo
 python-dateutil
 python-dotenv
 requests
+google-cloud-bigquery
```

Comment:
@123MwanjeMike this job requires `GOOGLE_APPLICATION_CREDENTIALS`. Please configure it (volumn mounts etc..) and update secret manager to point to the credentials file location.